package.json

```diff
@@ -7,9 +7,9 @@
   "chromium-args": "--disable-features=nw2",
   "default_locale": "en",
   "scripts": {
-    "start": "gulp debug",
+    "start": "NODE_ENV=development gulp debug",
```

Comment:
This breaks yarn start on windows 10 ... @chmelevskij pls can you check whats wrong

```
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Sorry, don't have access to windows. How do you set environment variables in windows usually?

Reply:
`set VAR=VALUE`